docs/validation.md

```
@@ -313,7 +327,9 @@ Passing tests alone is not enough if evidence, PR body, scope guard, or CI repor
 Before merge, review closeout must also confirm:
 
 - PR body includes evidence URL and commit hash.
-- Issue evidence comment exists.
+ - Issue evidence comment exists.
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)